src/model/defaults.ts

```diff
@@ -207,14 +215,16 @@ export const addDefaultModelFields = (model: Model, isNew: boolean): Model => {
  * @returns The model with default presets added.
  */
 export const addDefaultModelPresets = (list: Array<Model>, model: Model): Model => {
-  const defaultPresets: Array<ModelPreset> = [];
+  const defaultPresets: ModelEntityList<ModelPreset> = {};
 
   // Add default presets, which people can overwrite if they want to. Presets are
   // used to provide concise ways of writing advanced queries, by allowing for defining
   // complex queries inside the model definitions and re-using them across many
   // different queries in the codebase of an application.
-  for (const field of model.fields || []) {
-    if (field.type === 'link' && !field.slug.startsWith('ronin.')) {
+  for (const [fieldSlug, rest] of Object.entries(model.fields || {})) {
```

Comment:
To save on needing to call `Object.entries` on every loop interation.
```suggestion
  const fieldEntries = Object.entries(model.fields || {});
  for (const [fieldSlug, rest] of fieldEntries) {
```

Reply:
That's not how loops work in JS. The right part of the loop is only evaluated once.

Adding the separate variable IMO just adds unnecessary memory overhead.